ramalama/model.py

```diff
@@ -197,10 +197,10 @@ def gpu_args(self, force=False, server=False):
             or os.getenv("CUDA_VISIBLE_DEVICES")
             or platform.machine() in {"aarch64", "arm64"}  # linux and macOS report aarch64 differently
         ):
-            if server:
-                gpu_args += ["-ngl"]  # single dash
+            if runner:
+                gpu_args += ["--ngl"]  # single dash
```

Comment:
Comments are backwards.

Reply:
It's deliberate run is "--ngl" and everything else is "-ngl"

Reply:
Oh yes comments, fixing